crates/agglayer-storage/src/columns/nullifier_tree_per_network/mod.rs

```diff
@@ -0,0 +1,64 @@
+use bincode::Options;
+use serde::{Deserialize, Serialize};
+
+use super::{default_bincode_options, Codec, ColumnSchema, NULLIFIER_TREE_PER_NETWORK_CF};
+use crate::{error::Error, types::Hash};
+
+#[cfg(test)]
+mod tests;
+
+/// Column family for the nullifier tree per network.
+///
+/// | ------------ key ------------ |    | ------------- value -------------- |
+/// | NetworkId "root"              | => | hash(node.left) hash(node.right)   |
+/// | NetworkId hash(node)          | => | hash(node.left) hash(node.right)   |
+/// | NetworkId hash(node)          | => | hash(leaf)                         |
+/// | NetworkId "leaves" hash(leaf) | => | leaf bytes array                   |
```

Comment:
Same as for the balances, the nullifier tree leaves are boolean, so we could just store them directly.